sdc/ysdc_dataset_api/evaluation/metrics.py

```diff
@@ -176,14 +170,13 @@ def weighted_fde(ground_truth, predicted, weights, normalize_weights=False):
         per_plan_weights=weights)
 
 
-def _check_weights_are_non_negative(weights: np.ndarray):
-    if (weights < 0).sum() > 0:
-        raise ValueError('Weights are expected to be non-negative.')
-
-
-def _check_weight_sum(weights: np.ndarray):
-    if abs(weights.sum() - 1) > 1e-6:
-        raise ValueError('Weights are expected to sum to 1.')
+def log_likelihood(ground_truth, predicted, weights):
+    displacement_norms = np.sum((ground_truth - predicted) ** 2, axis=-1)
```

Comment:
I'd rather explicitly specify the value of sigma, which is assumed to be 1 here.

Otherwise seems legit.